modules/_labs/breadcrumbs/react/lib/Breadcrumbs/Dropdown/MenuItemLink.tsx

```diff
@@ -33,7 +33,7 @@ const MenuItemLink = styled('a')({
 export const DropdownMenuItemLink = forwardRef(
   (
     {children, href, ...elemProps}: DropdownMenuItemLinkProps,
-    ref: React.RefObject<HTMLAnchorElement>
+    ref: React.Ref<HTMLAnchorElement>
```

Comment:
This was caught by `strictFunctionTypes`

Reply:
whoops. 😬

Reply:
No runtime issue, just not the correct type. It was narrowed by the type override. Nothing depending on this `ref` required a `RefObject` or assumed it was. But it was nice `strictFunctionTypes` caught this as a potential problem.